src/contact/actions/Actions.js

```diff
@@ -0,0 +1,59 @@
+import * as ActionTypes from './ActionTypes';
```

Comment:
I am not a big fan of importing * - choose what you need or get an object will all you need. This approach is not a good practice in functional programming.

Reply:
Here I wanted all actions, thus imported all. But, I'll keep in mind this thing.